doc/api/errors.md

```diff
@@ -719,6 +719,19 @@ An attempt was made to register something that is not a function as an
 The type of an asynchronous resource was invalid. Users are also able
 to define their own types if using the public embedder API.
 
+<a id="ERR_BLOB_OPEN_FAILED"></a>
+
+### `ERR_BLOB_OPEN_FAILED`
+
+Opening a file path as a `Blob` failed.
+
+<a id="ERR_BLOB_READ_FAILED"></a>
+
+### `ERR_BLOB_READ_FAILED`
+
+Reading data from a `Blob` failed. This is typically due to the underlying
+storage of a file-backed `Blob` being modified on disc.
```

Comment:
```suggestion
storage of a file-backed `Blob` being modified on disk.
```

(That’s the spelling in the main fs docs and I think also just the more common spelling in general when referring to hard disks/file systems.)